### ISSUE TITLE
Some questions about code details

### ISSUE THREAD

**Issue by Xinchengzelin** [Tsinghua-MARS-Lab/DenseTNT#48]
I read the code on branch argoverse2,  could you help me answer some questions about the code?
1.  dataset_argoverse.py  L501  ,  the code should be like in the following?
```
lane_type_to_int[LaneType.BIKE] = 2 
lane_type_to_int[LaneType.BUS] = 3
```
2.  dataset_argoverse.py L527, why the code `vectors.append(vector)` not in the end of this for loop?
3.  decoder.py L339  `loss[i] += F.nll_loss(pred_probs[i].unsqueeze(0), torch.tensor([argmin], device=device))`, should the second parameter be `torch.tensor([1], device=device)`?
4.  dataset_argoverse.py L428  should add:
```
if len(focal_track.object_states) != 110:
            return None
```
    because some object_states of focal_track is less than 110, or the code won't run. 
5. argoverse2 didn't train the set_predictor, have you try to train set_predictor?  If the performance could be closet to that of offline optimization?

**Comment by GentleSmile** [comment]
1. Thanks for pointing it out!
2. This produces the same results as placing in the end of the loop
3. Why "1"? "Argmin" indicates the closest predicted trajectory.
4. If some object_states of focal_track is less than 110, there should be an error. But I have not seen this situation.
5. We will add set_predictor next week

**Comment by Xinchengzelin** [comment]
> 3\. nll_loss

3. It's my fault
4. Maybe there is someting wrong with my downloaded data, there are dozens of samples the object_states of focal_track is less than 110.  for example these 2 situations: 02ecafff-012c-45f8-bb6e-5ace2c6e3d88 and 014c02b0-3315-4faf-aca5-b239f5b7f0ca,  actually 105 frames data.
5. Looking forward to that

**Comment by GentleSmile** [comment]
We found that these two examples `02ecafff-012c-45f8-bb6e-5ace2c6e3d88` and `014c02b0-3315-4faf-aca5-b239f5b7f0ca` are both 110 frames.
